### ISSUE TITLE
Somethings FUCKED with spiritbreaker

### ISSUE THREAD

**Issue by Killette2** [d3athrow/vgstation13#14077]
1. eat ambrosia C on test server
2. expect to trip out insanely
3. nothing happens besides the space drug effect when I expect to hear 1000 screams a second

**Comment by JustSumBody** [comment]
Actually, I think hallucinations have been fucky recently in general.

With all the shardbombs I've seen this past week or two, I can't say I ever remember getting hallucinations from them. Someone brought this up yesterday and I didn't notice how odd it was.

**Comment by Killette2** [comment]
Yeah you're right, last time I remember a hallucination was around a week ago. Does this mean I can put mindbreaker in cryo and it be fine now as a filler chem?

**Comment by Allspice** [comment]
Spiritbreaker takes about 6 minutes and 9u to kick in under standard circumstances.

**Comment by ghost** [comment]
I saw a tajaran hallucination today, courtesy of somebody detonating the SME
